windshaft/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:vivid
+
+MAINTAINER Azavea
+
+RUN apt-get update
+
+# Windshaft requires node < 11; vivid has 10.25 in nodejs package
+RUN apt-get -q install -y nodejs npm git-core \
+    libcairo2-dev libpango1.0-dev libjpeg8-dev libgif-dev libpq-dev
+
+# link nodejs command to node
+RUN update-alternatives --install /usr/bin/node node /usr/bin/nodejs 10
+
+RUN mkdir -p /opt/windshaft
+WORKDIR /opt/windshaft
+COPY . /opt/windshaft
+
+# NB: the most recent versions do not work with the provided configurations
+RUN npm install windshaft@0.36.0
+
+EXPOSE 5000
+
+CMD ["node", "server.js"]
```

Comment:
Ok, will do. I'd doubt we'd want to bother with the Windshaft tests, as running them requires extra ImageMagick dependencies.